core/src/edu/rice/comp413fall2020grey/Location/registrar/ConnectionsManager.java

```diff
@@ -0,0 +1,13 @@
+package edu.rice.comp413fall2020grey.Location.registrar;
+
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ConnectionsManager {
```

Comment:
@yopyop1007  This looks like it has a lot of overlap with networking -- does this conflict with anything you have?

Reply:
I'm not sure, but I think they may be just creating a class through which they can send connectTo messages to the machines connected to the registrar (upon startup) to create a connected graph btw the super peers / connect the player clients to the super peers. Is this what the purpose of this was? If so, I don't think there's any overlap that would cause problems (maybe just a few refactors once we have implemented stub types)

Reply:
This was made under the impression that the registrar keeps track of the following information
Map<serverUUID, ip_address>
Map<serverUUID, List<clientUUID>>
and likely also 
Map<clientUUID, ip_address>
We are not clear how all the gRPC stuff will look like, this was made before we had decided on that.

Reply:
Sounds good to me. No need to make any changes on this file.